ImageBasedSimplification.ipynb

```diff
@@ -25,23 +25,6 @@
     "graph = ox.load_graphml('northeuroperails_alt.graphml')"
```

Comment:
<p>Please commit the output here, so that we don't have to run this to see what the edgelist is like</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/laitalaj/railanalysis/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>